deploy/olm-catalog/service-telemetry-operator/tests/scorecard/config.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Configuration
+apiversion: scorecard.operatorframework.io/v1alpha3
+metadata:
+  name: config
+stages:
+- parallel: true
+  tests:
+  - image: quay.io/operator-framework/scorecard-test:latest
+    entrypoint:
+    - scorecard-test
+    - basic-check-spec
+    labels:
+      suite: basic
+      test: basic-check-spec-test
+  - image: quay.io/operator-framework/scorecard-test:latest
+    entrypoint:
+    - scorecard-test
+    - olm-bundle-validation
+    labels:
+      suite: olm
+      test: olm-bundle-validation-test
```

Comment:
The contents of this scorecard file came from https://sdk.operatorframework.io/docs/testing-operators/scorecard/#config-file and matches the 2 tests we were encouraged to pass in STF-1256.